<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>32</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>18</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labPicture">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="actAddFolder"/>
    <addaction name="actAddFiles"/>
    <addaction name="separator"/>
    <addaction name="actScanItems"/>
    <addaction name="actDelItem"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFiles"/>
   <addaction name="actDelItem"/>
   <addaction name="actScanItems"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitWidth"/>
   <addaction name="actZoomFitHeight"/>
   <addaction name="actDockFloat"/>
   <addaction name="actDockVisible"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddFolder">
   <property name="text">
    <string>添加目录...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="text">
    <string>添加文件...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="text">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actZoomFitWidth">
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actDelItem">
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actZoomFitHeight">
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="text">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
